transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py

```diff
@@ -850,19 +886,34 @@ def get_filtered_dataset(
     # converting TIME_PERIOD to numeric: we should get integers by default
```

Comment:
@jamescw: could we simplify and set `dtype=str` at `line 867`? This could make our life easier:
- Avoid the requirement of setting `DTYPE` at indicators config level for those with `YES/NO` values
- Eventually, we are taking care of proper `conversion to_numeric` for `OBS_VALUE` and `TIME_PERIOD` at `lines 876-887`
- Would also reduce the requirement to convert `astype()` in actual `lines 892-897`